python/dataprep.py

```diff
@@ -12,6 +12,7 @@
 from keras.utils import to_categorical
 from sklearn.exceptions import DataConversionWarning
 import warnings
+from scipy import sparse
```

Comment:
If this is need for the previous commit, I'd squash this commit in to the previous one, or move it before it.

Reply:
I think I just did it in an interactive rebase. Just checking now...

Reply:
it's gone. Hurrah. Did fixup instead of squash...